2/contrib/jenkins/install-plugins.sh

```diff
@@ -151,6 +151,12 @@ BUNDLE_PLUGINS=${BUNDLE_PLUGINS:-/opt/openshift/plugins/bundle-plugins.txt}
 JENKINS_WAR=${JENKINS_WAR:-/usr/lib/jenkins/jenkins.war}
 JENKINS_UC=${JENKINS_UC:-https://updates.jenkins.io}
 
+if [ ! -f $JENKINS_WAR ]; then
+  JENKINS_WAR=/usr/share/java/jenkins.war
+  mkdir -p /usr/lib/jenkins/
+  ln -s $JENKINS_WAR /usr/lib/jenkins/jenkins.war
+fi
```

Comment:
Why is this now required?

Reply:
The file location was changed, so this should make things backwards compatible.